core/App/screens/Home.tsx

```diff
@@ -1,5 +1,4 @@
-import { CredentialExchangeRecord as CredentialRecord, CredentialState } from '@aries-framework/core'
-import { useCredentialByState } from '@aries-framework/react-hooks'
+import { CredentialExchangeRecord as CredentialRecord } from '@aries-framework/core'
```

Comment:
This is a good opportunity to switch out `CredentialRecord` for `CredentialExchangeRecord` to cleanup the imports.

Reply:
I've created #391 to track this.  AFJ's `'@aries-framework/core'` doesn't have `CredentialRecord` and it would touch quite a few files unrelated to this PR. So not sure where to go with the change.